test/models/king_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class KingTest < ActiveSupport::TestCase
+
+	##### TEST VALID_MOVE METHOD ######
+
+  	test "check if move is valid (invalid case, no movement)" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		king = game.pieces.where(:pos_x => 3).where(:pos_y => 0).first
+		move = king.valid_move?([3, 0])
+		assert_not move	
+	end
+
+	test "check if move is valid (invalid case, off board)" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
```

Comment:
Hm, I just ran the tests on my machine again, went into the rails console and viewed Game.all... I only see the items I created from the console earlier. I don't see any of the games created from the test, so I think it is deleting them after, unless I am misunderstanding what you are saying?